### PR TITLE
ONYX-8600: Add Authn-JWT-Configuration tests

### PR DESCRIPTION
**Pull request by tzheleznyak**
### What does this PR do?
Tests for configuration checks in jwt authenticator

### What ticket does this PR close?
ONYX-8600

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [x] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

#### API Changes
- [ ] The [OpenAPI spec](https://github.com/cyberark/conjur-openapi-spec) has been updated to meet new API changes (or an issue has been opened), or
- [x] The changes in this PR do not affect the Conjur API

### CODE REVIEW

File: cucumber/authenticators_jwt/features/authn_jwt_configuration.feature [hunk 1]
```diff
@@ -0,0 +1,146 @@
+Feature: JWT Authenticator - Configuration Check
+  Background:
+    Given I initialize JWKs endpoint with file "myJWKs.json"
```
Comment (Avi-Eyal) [inline cucumber/authenticators_jwt/features/authn_jwt_configuration.feature:3]: JKWs => JWKS

Comment (tzheleznyak) [reply cucumber/authenticators_jwt/features/authn_jwt_configuration.feature]: Done

**Review by Avi-Eyal — APPROVED** [review]
LGTM

**Review by Avi-Eyal — APPROVED** [review]
🚀

**Pull request merged**